sdf.md

```diff
@@ -642,18 +642,22 @@ All names in SDF are case-sensitive.
 
 # SDF structure
 
-SDF definitions are contained in SDF files.  One or more SDF files can
-work together to provide the definitions and declarations that are the
-payload of the SDF format.
+SDF definitions are contained in SDF documents, together with data
+about the SDF document itself (information block).
+Definitions and declarations from additional SDF documents can be
```

Comment:
This sentence ended up a bit complex. Would it be simpler to say some thing like "Using references an SDF document can use definitions and declarations from other SDF documents to build models.".

Reply:
Yes, but models are also built when other documents are not referenced (the point here is how models get to be).
I have simplified the sentence slightly without losing this point...